glean-core/src/upload/directory.rs

```diff
@@ -14,7 +14,9 @@ use serde_json::Value as JsonValue;
 use uuid::Uuid;
 
 use super::PingRequest;
-use crate::Result;
+
+pub const PENDING_PINGS_DIRECTORY: &str = "pending_pings";
+pub const DELETION_REQUEST_PINGS_DIRECTORY: &str = "deletion_request";
```

Comment:
These values are also used in the `ping` module, where they are hardcoded. Would it make sense to have these constants be a shared on the crate level? So that any module can access them?

Reply:
Yes.